scripts/docker/build.sh

```diff
@@ -123,7 +123,19 @@ case "${SERVICE}" in
         DOCKER_CONTEXT="dockerfiles/"
         ;;
     mina-toolchain)
-        DOCKERFILE_PATH="dockerfiles/stages/1-build-deps dockerfiles/stages/2-opam-deps dockerfiles/stages/3-toolchain"
+        DOCKERFILE_PATH_SCRIPT_1="dockerfiles/stages/1-build-deps"
+        DOCKERFILE_PATH_SCRIPT_2_AND_MORE="dockerfiles/stages/2-opam-deps dockerfiles/stages/3-toolchain"
+        case "${INPUT_CODENAME}" in 
+          bullseye|focal)
+            DOCKERFILE_PATH="$DOCKERFILE_PATH_SCRIPT_1 dockerfiles/stages/1-build-deps-bullseye-and-focal $DOCKERFILE_PATH_SCRIPT_2_AND_MORE"
+            ;;
+          noble)
+            DOCKERFILE_PATH="$DOCKERFILE_PATH_SCRIPT_1 dockerfiles/stages/1-build-deps-noble $DOCKERFILE_PATH_SCRIPT_2_AND_MORE"
+            ;;
+          *)
+            DOCKERFILE_PATH="$DOCKERFILE_PATH_SCRIPT_1 $DOCKERFILE_PATH_SCRIPT_2_AND_MORE"
```

Comment:
I suggest we just panic in this case. Without ssl system dependency we just can't compile MINA